### PR TITLE
Implement CI/CD Pipeline

### PR DESCRIPTION
**Pull request by kamaldodiya**

### CODE REVIEW

File: Dockerfile.prod [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM nginx:1.14
+
+EXPOSE 80 9000
```
Comment (azangru) [inline Dockerfile.prod]: is anything listening to port 9000 in this container? Why are we exposing it?

File: Dockerfile.prod [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+RUN rm /etc/nginx/conf.d/*
```
Comment (azangru) [inline Dockerfile.prod]: this shouldn't be necessary. We only want to overwrite the default.conf file in the /etc/nginx/conf.d, which is what the next line is doing.

File: Dockerfile.prod (hunk 1 above)
Comment (kamaldodiya) [reply Dockerfile.prod]: Removed 9000 port. Don't know why it was there

File: Dockerfile.prod (hunk 2 above)
Comment (kamaldodiya) [reply Dockerfile.prod]: Corrected this one as well...

**Review by azangru — APPROVED** [review]

**Pull request merged**